### PR TITLE
REP-4668 - Versioned docs: navigation and scaffolding pages

### PR DESCRIPTION
**Pull request by Mario-Lopez**
Versioned docs - navigation and scaffolding pages

Changes include:
* updates to the gradle build
* home page for versioned docs
* pages for three filters

### CODE REVIEW

File: versions.properties [hunk 1]
```diff
@@ -43,6 +43,7 @@ net.sf.ehcache:ehcache-core=2.6.0
 saxonVersion=9.5.1-8
 net.sf.saxon:Saxon-EE=$saxonVersion
 abderaVersion=1.1.3
+org.asciidoctor:asciidoctorj-diagram=1.5.0
```
Comment (adrianjgeorge) [inline versions.properties]: Even if it did come before apache alphabetically, getting in between the abdera declarations and the abdera version number is just bad.

Comment (dmnjohns) [inline versions.properties]: It was inserted alphabetically. Fair enough, though, I'll move it after the apache stuff.

Comment (adrianjgeorge) [inline versions.properties]: do you even alphabet bro?

File: repose-aggregator/build.gradle [hunk 2]
```diff
@@ -3,8 +3,23 @@ import java.text.SimpleDateFormat
...
+    asciidoctor 'org.asciidoctor:asciidoctorj-diagram'
+}
+
 asciidoctor {
-    attributes 'source-highlighter': 'coderay'
+    requires 'asciidoctor-diagram'
+    attributes 'source-highlighter': 'coderay',
+               'stylesdir': "${project.projectDir}/src/docs/asciidoc/_stylesheets",
+               'stylesheet': 'repose.css',
+               'imagesoutdir': "${project.buildDir}/asciidoc/html5/images",
```
Comment (Mario-Lopez) [inline repose-aggregator/build.gradle]: I see you already fixed it on the REP-4712 branch, but I'll put it out there that this line breaks the diagram images.

Comment (dmnjohns) [inline repose-aggregator/build.gradle]: Indeed. When I initially added it, the generated images were already cached in the build directory, so I didn't catch it at first.

I'm kind of annoyed, actually. From what I can tell, while you can specify the output directory, there isn't a way to tell asciidoctor-diagram to use that as the input directory for the image. I wanted it to be smarter. 😞

**Comment by Mario-Lopez** [comment]
Team approved pull request.  Any remaining issues will be fixed in REP-4712.

**Pull request merged**